component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/util/Constants.java

```diff
@@ -112,4 +112,5 @@ public class Constants {
     public static final String MOESIF_CONTENT_TYPE_HEADER = "application/json";
     public static final String MOESIF_CONTENT_TYPE_KEY = "Content-Type";
     public static final String MOESIF_USER_AGENT_KEY = "User-Agent";
+    public static final String X_ORIGINAL_GW_URL = "x-original-gw-url";
```

Comment:
Change the name of the constant.

Reply:
Resolved with: 090a6542c6d266c72017469a67284685581b60ea